### PR TITLE
Update eslint-config-airbnb to version 11.1.0

### PR DESCRIPTION
**Pull request by victorporof**
Greenkeeper has been trying to update our airbnb config for quite a while now, but we kept deleting those PRs: https://github.com/mozilla/tofino/pull/1145. It's about time we did something about this.

### CODE REVIEW

File: test/unit/user-agent-service/test-visited.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const { DB_ADD, TEMPIDS } = helpers;
 const perfTesting = false;
 
 describe('Storage.recentlyStarred', () => {
-  it('Stars and unstars.', async () => {
+  it('Stars and unstars.', (async) () => {
```
Comment (Mossop) [inline test/unit/user-agent-service/test-visited.js]: This doesn't look right, there are a few other instances throughout too.

**Comment by victorporof** [comment]
Jordan can you please take a quick look at the frontend changes?

**Comment by victorporof** [comment]
@Mossop please take a look at this again.

File: test/unit/user-agent-service/test-visited.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const { DB_ADD, TEMPIDS } = helpers;
 const perfTesting = false;
 
 describe('Storage.recentlyStarred', () => {
-  it('Stars and unstars.', async () => {
+  it('Stars and unstars.', async function() {
```
Comment (Mossop) [inline test/unit/user-agent-service/test-visited.js]: It seems like it's a bug in the rule if this is needed. Fine to take it like this but is it worth checking if it is reported upstream?

Comment (victorporof) [inline test/unit/user-agent-service/test-visited.js]: Will report it upstream. I twas definitely a bug to wrap `async` in parens.

**Comment by jsantell** [comment]
This requires no parens in single-arg arrow functions, and no async arrow functions? I don't see the benefits for this strictness, and my opinions of death-by-a-thousand-paper-cuts on our many strict linting rules are well known, but the code changes look fine.

**Pull request closed without merging**